documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -41,44 +41,65 @@ include::modules/about-cold-warm-migration.adoc[leveloffset=+2]
 
 Review the following prerequisites to ensure that your environment is prepared for migration.
 
-[id="software-requirements_{context}"]
-=== Software requirements
+These prerequisites apply to all migrations. Prerequisites that apply only to a specific provider are discussed in the section dedicated to that source provider.
```

Comment:
@nunzy1 I think I'll leave this as is because we are switching the structure of the user guide -- after the general prerequisites and installation instructions, for each provider we'll have a separate section including prerequisites, UI instructions, and CLI instructions. Does my phrasing in the PR make more sense now?